### ISSUE TITLE
ERROR - Logs Account

### ISSUE THREAD

**Issue by aboutandthere** [farshadz1997/Microsoft-Rewards-bot#28]
Hi am using Chromebook (Linux Debian)

When trying to run the script, i encounter the following error:

[LOGS] "Logs_accounts.txt" file not found.
Traceback (most recent call last):
  File "/home/recoveryfarmer02/Microsoft-Rewards-bot-master/ms_rewards_farmer.py", line 1031, in logs
    LOGS = json.load(open(f"Logs_{filename}.txt", "r"))
FileNotFoundError: [Errno 2] No such file or directory: 'Logs_accounts.txt'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/recoveryfarmer02/Microsoft-Rewards-bot-master/ms_rewards_farmer.py", line 1273, in <module>
    main()
  File "/home/recoveryfarmer02/Microsoft-Rewards-bot-master/ms_rewards_farmer.py", line 1256, in main
    logs()
  File "/home/recoveryfarmer02/Microsoft-Rewards-bot-master/ms_rewards_farmer.py", line 1071, in logs
    updateLogs()
  File "/home/recoveryfarmer02/Microsoft-Rewards-bot-master/ms_rewards_farmer.py", line 1076, in updateLogs
    with open(f'Logs_{filename}.txt', 'w') as file:
PermissionError: [Errno 13] Permission denied: 'Logs_accounts.txt'

Can you please kindly help?

Thank you.

**Comment by farshadz1997** [comment]
Hi,
It looks like your system does not let bot access bot directory for creating logs, I'll look how can I resolve your issue.

**Comment by aboutandthere** [comment]
Thanks for the quick reply - looking forward to a solution!

**Comment by aboutandthere** [comment]
> Hi, It looks like your system does not let bot access bot directory for creating logs, I'll look how can I resolve your issue.

Hi again, I have used "sudo chmod 'USER' Logs_accounts.txt" which has let the script run past the original error, but now it is saying I'm on wrong version of Chromedriver. I have checked and I've downloaded the correct version for my Chromium (105). 

[LOGS] Logs loaded successfully.

********************Your Email********************
[Driver] Session not created.
[Driver] Please download correct version of webdriver form link below:
[Driver] https://chromedriver.chromium.org/downloads

**Comment by aboutandthere** [comment]
Hi again - apologies for the extra comments.

I can confirm that I have now resolved the problem by doing the following:

First I made 'Logs_accounts.txt' readable and writeable by doing $ sudo chown 'USER HERE' Logs_accounts.txt

This let the script run, but produced the error that I was on wrong chromedriver. (I had checked and I had downloaded the right one - I am on 105).

I then restarted my PC and upon loading up, I deleted all findable versions and files of chromedriver.

I then did $ ls -l

This showed me a list of chromedriver zip files I had made root.

I then did $ sudo rm chromedriver_linux64.zip.1 (.2, .3 etc until I had removed all versions I could see).

I then did $ sudo wget[ ](https://chromedriver.storage.googleapis.com/index.html?path=105.0.5195.52/)

Followed by $ sudo unzip chromedriver_linux64.zip

$ sudo mv chromedriver /usr/bin/chromedriver

$ sudo chown root:root /usr/bin/chromedriver

$ sudo chmod +x /usr/bin/chromedriver

It seems this process has worked for me :)
